specification/storage/data-plane/Microsoft.FileStorage/preview/2019-02-02/file.json

```diff
@@ -4214,6 +4214,7 @@
             "description": "Only update is supported: - Update: Writes the bytes downloaded from the source url into the specified range.",
             "required": true,
             "type": "string",
+            "x-ms-parameter-location": "method"
             "enum": [
                 "update"
             ],

```

Comment:
This is invalid json, right?  Don't you need a comma after the location and is the diff saying you deleted the colon?

Reply:
Aha, I was wondering what the red spot trying to tell me!  Fixed.

Reply:
Ohhhh, that's what the hell the weird red colon was?  I couldn't figure that out either.